src/backend/templates/schedule.hbs

```diff
@@ -1041,6 +1052,7 @@
         if(isCalendarView) {
           viewsFilter('calendar', className);
         } else {
+          console.log(className);
```

Comment:
@agbilotia1998 was this log statement intentional or for testing?

Reply:
Removed, thanks for pointing out :sweat_smile: